src/faces.jl

```diff
@@ -0,0 +1,122 @@
+# Create recipe: 
+@Makie.recipe(Faces, grid) do scene
+    Makie.Attributes(;
+    
+      # Use default theme:
+      Makie.default_theme(scene, Makie.Mesh)...,
+      Makie.default_theme(scene, Makie.Lines)...,
+      Makie.default_theme(scene, Makie.Scatter)...,
+
+      # Custom arguments:
+      color      = :pink,
+      colormap   = :bluesreds,
+      colorrange = nothing,
+      fieldstyle = :nodes,
+
+      # Otherwise, 2D plots don't use custom default colors:
+      cycle      = nothing,
+
+      # Force no shading:
+      shading    = false
+    )
+end
+  
+# Plot! especialization:  
+function Makie.plot!(plot::Faces{<:Tuple{Grid}})
+
+  grid = plot[:grid][]
+  Dc = num_cell_dims(grid)
+
+  # Dispatch edges according to cell dimension Dc:
+  if Dc == 0
+    plot_Ngon_vertices!(plot, grid)
+  elseif Dc == 1
+    plot_Ngon_edges!(plot, grid)
+  else
+    plot_Ngon_faces!(plot, grid |> to_face_grid)
+  end
+
+end
+
+# 2D plots:
+function plot_Ngon_faces!(plot::Faces{<:Tuple{Grid}}, grid::Grid)
+
+  # Retrieve plot attributes:
+  fieldstyle    = plot[:fieldstyle][]
+  color         = plot[:color][]
+
+  grid_ = to_plot_mesh(grid)
+  if color isa AbstractVector
+    if plot[:colorrange][] === nothing
+      plot[:colorrange][] = extrema(color)
+    end
+    if length(color) != GeometryBasics.coordinates(grid_) |> length
```

Comment:
`color` is handled differently in `plot_Ngon_faces!`, `plot_Ngon_edges!`, and `plot_Ngon_vertices!`. This needs to be unified. Move all the handling of `color` to a helper function and use it when necessary.

Reply:
In addition, I don't know if this will work in reactive animations.